modules/microshift-adding-service-to-blueprint.adoc

```diff
@@ -37,7 +37,7 @@ enabled = ["microshift"]
 EOF
 ----
 [.small]
-<1> Optional `microshift-greenboot` RPM. For more information, read the "Greenboot health check" guide in the "Running Applications" section.
+<1> The `microshift-greenboot` RPM. For more information, read the "Greenboot health check" guide in the "Running Applications" section.
```

Comment:
I think lines 31-33 should be removed together with the note. When `microshift` RPM is installed, it pulls greenboot automatically, alongside other RPM dependencies.

Reply:
Removed in the bug PR (by accident), so there we go! https://github.com/openshift/openshift-docs/pull/63057